src/statego-backend/.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "rust-analyzer.linkedProjects": [
+        ".\\Cargo.toml"
```

Comment:
could you add "./Cago.toml" as well, that will make it work for non windows systems